### ISSUE TITLE
Add running examples to be incorporated as tests

### ISSUE THREAD

**Issue by martin-neugebauer** [SPF-OST/pytrnsys_gui#236]
Example projects should be added that can successfully be launched and post-processed from the GUI. These will be incorporated as tests to ensure the post-processed results are not changing when changes are made to the source code. The following examples should be included:
- [ ] TRI-HP dual source
- [ ] TRI-HP slurry
- [ ] PvAshpDhwShMfh @jschmidl 
- [ ] StAshpDhwShMfh @jschmidl

**Comment by martin-neugebauer** [comment]
@zuckerruebe @jschmidl @mschu84 I'll go first with the dual source and provide you with details on how to set up your example once I completed it.

**Comment by martin-neugebauer** [comment]
@jschmidl You can add which examples you'll provide by editing the post at the top.

**Comment by mschu84** [comment]
@martin-neugebauer what about defining a list of values to be printed for comparison? (even if we compare postprocessed annual values we have to make sure that we print everything that should be compared - another idea would be a "comparison - printer" printing all important things into the same file and this can be compared in order to not need the postprocessing step?) We can add this later but when we are anyway preparing the examples now would be less effort than adding it later?

**Comment by martin-neugebauer** [comment]
Yes, I thought of comparing prt-files originally, but @danielcarbonell had some concerns about that (I am not sure anymore what they were in detail). I'd prefer to not use any post-processing, since this is a pure pytrnsys-thing. I don't know how the "comparison - printer" would work... Normally our tests compare an expected to an actual file.

In any case we will have to do some rounding to compare on a meaningful uncertainty level.

**Comment by zuckerruebe** [comment]
@jschmidl Can you check the boxes at the top if appropriate?

**Comment by jschmidl** [comment]
> @jschmidl Can you check the boxes at the top if appropriate?

PvAshpDhwShMfh was running when i implemented it... though no test was created
